### PR TITLE
Make driver-container-base simpler

### PR DESCRIPTION
**Pull request by zvonkok**
This PR simpilifies the handling of repos and rpms in driver-container-base. 
The simple-kmod is now systemd based.

### CODE REVIEW

File: config/recipes/simple-kmod/manifests/1000-driver-container.yaml [hunk 1]
```diff
@@ -122,7 +122,8 @@ spec:
       - image: image-registry.openshift-image-registry.svc:5000/{{.SpecialResource.Spec.Metadata.Namespace}}/{{.SpecialResource.Name}}-{{.GroupName.DriverContainer}}:v{{.KernelVersion}}
         imagePullPolicy: Always
         name: {{.SpecialResource.Name}}-{{.GroupName.DriverContainer}}-{{.OperatingSystemMajor}}
-        command: ["/bin/entrypoint.sh"]
+#        command: ["/bin/entrypoint.sh"]
+        command: ["/sbin/init"]
```
Comment (ArangoGutierrez) [inline config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: I don't understand this change, the configMap and the way is mounted is still called `/bin/entrypoint.sh` , can't see the file of `/sbin/init`

**Review by ArangoGutierrez — COMMENTED** [review]
Looks good, just one comment

File: config/recipes/simple-kmod/manifests/1000-driver-container.yaml (hunk 1 above)
Comment (zvonkok) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: Ooops, need to delete the configMap stuff.

Comment (zvonkok) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: `/sbin/init` is systemd

Comment (zvonkok) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: With systemd and unit files we do not have to do the "stupid" sleep infinity stuff to the driver-containers and it is inherent part of KVC.

Comment (zvonkok) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: The internal KVC update will come with another PR.

Comment (ArangoGutierrez) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: will you rm the ConfigMap in this PR or in a following one?

Comment (zvonkok) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: In this. Will update soon.

Comment (ArangoGutierrez) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: once you push the change, I will drop the lgtm ;)

Comment (ArangoGutierrez) [reply config/recipes/simple-kmod/manifests/1000-driver-container.yaml:66]: the new commits don't address this comment

**Review by ArangoGutierrez — APPROVED** [review]
Thanks for this PR @zvonkok 
/lgtm

**Pull request merged**